sync2/rangesync/rangesync.go

```diff
@@ -38,6 +38,23 @@ type RangeSetReconcilerConfig struct {
 	MessageLimit int `mapstructure:"message-limit"`
 }
 
+func (cfg *RangeSetReconcilerConfig) Validate() error {
+	var errs []error
+	if cfg.MaxSendRange <= 0 {
+		errs = append(errs, errors.New("max-send-range must be positive"))
+	}
+	if cfg.ItemChunkSize == 0 {
+		errs = append(errs, errors.New("item-chunk-size must be positive"))
+	}
+	if cfg.SampleSize > maxSampleSize {
+		return fmt.Errorf("bad sample-size %d (max %d)", cfg.SampleSize, maxSampleSize)
```

Comment:
Why is this the only one that is returned by itself and not joined?

Reply:
It was done by mistake. Fixed